hack/tools/lifted/lifted.go

```diff
@@ -0,0 +1,243 @@
+package main
```

Comment:
What do you think about the name `lifted-gen`? Just like `client-gen`/`deepcopy-gen`? :)

`hack/tools/lifted-gen/main.go`.